packages/flame_audio/lib/audio_pool.dart

```diff
@@ -14,8 +15,13 @@ class AudioPool {
   final Map<String, AudioPlayer> _currentPlayers = {};
   final List<AudioPlayer> _availablePlayers = [];
 
+  /// The path of the sound of this pool
   final String sound;
+
+  /// If the poop is repeating
```

Comment:
Hahah, I hate when the poop is repeating!
```suggestion
  /// If the pool is repeating
```